frame/evm/src/lib.rs

```diff
@@ -534,4 +534,32 @@ impl<T: Trait> Module<T> {
 			Self::remove_account(address);
 		}
 	}
+
+	/// Withdraw fee.
+	pub fn withdraw_fee(address: &H160, value: U256) {
```

Comment:
Is `withdraw_fee ` some additional trait from evm pallet? Looks good to me, just curious why it need an extra fee specific mutate api

Reply:
Here is just a simple encapsulation of balance modification.